gateway/1.7/modules/ROOT/nav.adoc

```diff
@@ -114,6 +114,7 @@
 * xref:flex-troubleshoot.adoc[Troubleshooting]
 ** xref:flex-troubleshoot-debug-logs.adoc[Troubleshoot with Debug Logs]
 ** xref:flex-troubleshoot-reg.adoc[Troubleshoot Registration Issues]
+** xref:flex-troubleshoot-platform-connection.adoc[Troubleshoot Platform Connection]
```

Comment:
This should match the topic heading, which has a gerund - "Troubleshooting."
I also think that this should be "Troubleshooting Platform Connections."